### PR TITLE
added students checklist

### PR DESCRIPTION
**Pull request by Nattalli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,9 +45,10 @@ Create `sidebar.html` there. Inside `sidebar.html`:
...
         - Drivers
     - In `base.html` include `sidebar.html`, so all these links will be accessible on all pages.
-9. Run server, open `http://127.0.0.1:8000/`, check if the information is there and if it is correct.
-10. Run `python manage.py test` to check your code results.
+9. Check that you put empty lines at the end of each HTML file.
+10. Run server, open `http://127.0.0.1:8000/`, check if the information is there and if it is correct.
+11. Run `python manage.py test` to check your code results.
 
-NOTE: Attach screenshots of all created or modified pages to pull request. It's important to attach images not links to them. See example:
+### Note: Attach screenshots of all created or modified pages to pull request. It's important to attach images not links to them. See example:
```
Comment (DmytroSvirsa) [inline README.md:53]: and somewhere - make sure that you don't push 5000+ files

**Review by DmytroSvirsa — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Abnormaltype) [reply README.md:53]: good point

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Nattalli) [reply README.md:53]: @DmytroSvirsa really I don't understand what you mean. Can you give an example of what is it about?

File: README.md [hunk 2]
```diff
@@ -44,9 +45,10 @@ Create `sidebar.html` there. Inside `sidebar.html`:
...
```
[the remark this answers is not in the document]
Comment (Nattalli) [reply README.md:52]: btw, in the example, Polina attached a screenshot in the commit, doesn't she?

[the remark this answers is not in the document]
Comment (Abnormaltype) [reply README.md:52]: maybe not at all tasks

**Review by Abnormaltype — APPROVED** [review]
LGTM

**Review by DmytroSvirsa — APPROVED** [review]

**Pull request merged**